chain-vote/src/lib.rs

```diff
@@ -242,12 +290,25 @@ fn group_elements_from_bytes(bytes: &[u8]) -> Option<Vec<gang::GroupElement>> {
     Some(elements)
 }
 
+pub fn verify_decrypt_share(
```

Comment:
yes, good point. I was thinking of changing the `lib.rs` file, and try to keep consistency with what we have in the rest of the library (basically, just defining the modules). I initially left it out of this PR because the goal was to compare against `decrypt_nizk`, but given that we are now doing the PR against `master`, do you think it makes sense to do this refactor here?

Reply:
Yep, it does. Just lay out whatever you think is the best structure

Reply:
Ok, some major refactor coming then

Reply:
agreed to put it in a different PR